### DISCUSSION TITLE
Welcome to dxcharts-lite Discussions!

### DISCUSSION THREAD

**Discussion by Alex-Kostusenko** [devexperts/dxcharts-lite#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luca-giuranna** [comment]
I would like to ask if this lite version supports trading from charts. I plan to use it to build a personal trading app. Thanks.
